webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const webpack = require('webpack');
```

Comment:
You should have a separate config for dev and prod. I'll suggest you create 3 files, one will be the base file that will contain shared settings between dev and prod and the other 2 will contain the environment specific config.

One example is devServer, you only need it for development and not on production. Also, you might need to add some optimization plugins specifically for production.

Reply:
I have done that but I'm not sure of the optimization plugins for production